BUILD.bazel

```diff
@@ -73,13 +73,29 @@ proto_plugin(
     visibility = ["//visibility:public"],
 )
 
+proto_library(
```

Comment:
I no longer work at Astranis, but we needed the BUILD file to use the google protobuf instead of a separate protobuf for python. Otherwise, you end up with duplicate modules defining the same messages and it won't work at runtime. Please revert this unless it's necessary

Reply:
interestingly, I found there are two `py_proto_library` commonly used. One is from `protobuf` repo, the other one is from `rules_python`. For the one in `protobuf`, I have to build python proto separately from source instead of using built "descriptor_proto" target. But the one from `rules_python` can do it. I switched to use it and make it much cleaner here.

Reply:
Yeah, the `rules_python` version is the future. The `protobuf` implementation was just a stop-gap and will eventually be removed.